StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/MandateTextProvider.swift

```diff
@@ -10,6 +10,8 @@ import Foundation
 @_spi(STP) import StripeUICore
 
 protocol MandateTextProvider {
+    var merchantDisplaysMandate: Bool { get }
```

Comment:
delte this, the computed property in implemntation, and `hidesMandateText` IMO.

Reply:
Deleted!